EasyVCR.Tests/CensorsTest.cs

```diff
@@ -0,0 +1,165 @@
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyVCR.Tests
+{
+    [TestClass]
+    public class CensorsTest
+    {
+        [TestMethod]
+        public void TestDefaultCensors()
+        {
+            var censors = Censors.Default;
+
+            Assert.IsTrue(true);
+        }
+
+        [TestMethod]
+        public void TestApplyBodyParametersCensorsNoContentType()
+        {
+            const string body = "{\"foo\":\"bar\"}";
+            InternalUtilities.ContentType? contentType = null;
+
+            var censors = new Censors();
+
+            Assert.ThrowsException<VCRException>(() => censors.ApplyBodyParametersCensors(body, contentType));
```

Comment:
question: why do we expect an error here?

hint: docstring could easily fix this question